docs/core/getting-started.md

```diff
@@ -14,45 +14,23 @@ ms.assetid: d1453a0d-317c-4da6-b68e-422a2c0bfe49
 
 # Getting started with .NET Core
 
-.NET Core runs on [Windows](#windows), [Linux](#linux), and [macOS / OS X](#os-x--macos).
-
-## Windows
-
-Install .NET Core on [Windows](https://www.microsoft.com/net/core#windows). 
-
-You can get started developing .NET Core apps by following these step-by-step tutorials.
-
-* [Getting started with .NET Core on Windows, using Visual Studio 2015](tutorials/using-on-windows.md) - Use [Visual Studio](https://www.visualstudio.com/), the full-featured integrated development environment (IDE) for Windows.  
-* [Getting started with .NET Core on macOS, using Visual Studio Code](tutorials/using-on-macos.md) - Use [Visual Studio Code](https://www.visualstudio.com/products/code-vs), Microsoft's lightweight cross-platform code editor. This tutorial is written for macOS, but Visual Studio Code also works on Windows.
-* [Getting started with .NET Core using the command line](tutorials/using-with-xplat-cli.md) - Use any code editor with the [.NET Core cross-platform command-line interface (CLI)](tools/index.md).
-
-See the [Prerequisites for Windows development](windows-prerequisites.md) topic for a list of the supported Windows versions. 
-
-## Linux
-
-Install Linux on your distribution/version:
+.NET Core runs on [Windows](windows-prerequisites.md), Linux, and macOS / OS X. Install it on your operating system using the following links:
 
+* [Windows 7 - 10 & Server 2008 - 2016](https://www.microsoft.com/net/core#windows)
```

Comment:
Maybe change it to:

* [Windows 7 SP1, 8.1, 10 & Windows Server 2008 - 2016](https://www.microsoft.com/net/core#windows) (Read the [Prerequisites for Windows development](windows-prerequisites.md) topic for more details.)

    > OR: Read the [Prerequisites for Windows development](windows-prerequisites.md) topic for more details.